### PR TITLE
Add a Fragment capable of adding support of a PostgreSQL database to a Spring Boot based application.

### PR DESCRIPTION
**Pull request by dmbelyaev**
Co-authored-by: Dmitry Belyaev <dbelyaev@vmware.com>

### CODE REVIEW

File: spring-boot-postgresql/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Then in your `engine` section add an `InvokeFragment` directive at an appropriate level.
+
+```yaml
+engine:
+  # ...
+    - type: InvokeFragment
+      reference: spring-boot-postgresql
+      let:
+        - name: todododo
```
Comment (ericbottard) [inline spring-boot-postgresql/README.md]: I assume this was written early and you forgot to revisit?

File: spring-boot-postgresql/accelerator.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
+accelerator:
+  options:
+    - name: databaseName
+      inputType: text
+      defaultValue: "postgres-database"
+      label: Database Instance Name this Application will use (can be existing one in the cluster)
```
Comment (ericbottard) [inline spring-boot-postgresql/accelerator.yaml]: I see you're often using `label` to express long explanations of what options do. I _think_ `description` could be more appropriate. Please test and let me know what you think.

File: spring-boot-postgresql/accelerator.yaml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            with: |
+              '${startOfDependencies}
+                  implementation("org.testcontainers:postgresql:1.17.3")${existingDependencies}'
+
+    - include: [ "README.md" ]
+      chain:
+        - merge:
+            - type: ReplaceText # No-op to trick the engine into thinking there IS a merge
+          onConflict: FavorForeign
+        - type: ReplaceText
```
Comment (ericbottard) [inline spring-boot-postgresql/accelerator.yaml:126]: So you didn't extract that in a file after all?

File: spring-boot-postgresql/README.md (hunk 1 above)
Comment (ext-jaret) [reply spring-boot-postgresql/README.md]: Yup.... fixed. Thanks.

File: spring-boot-postgresql/accelerator.yaml (hunk 2 above)
Comment (ext-jaret) [reply spring-boot-postgresql/accelerator.yaml]: Indeed, changed long sentence to `description` but kept the label with short readable lable name.

File: spring-boot-postgresql/accelerator.yaml (hunk 3 above)
Comment (dmbelyaev) [reply spring-boot-postgresql/accelerator.yaml:126]: No, we didn't.

**Review by trisberg — APPROVED** [review]

**Pull request merged**